README.md

```diff
@@ -170,6 +171,10 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [turkish-number](https://github.com/izniburak/turkish-number) - Turn integers into the Turkish words
  * [wkhtmltopdf-crystal](https://github.com/blocknotes/wkhtmltopdf-crystal) - Bindings / wrapper for libwkhtmltox (HTML to PDF / image converter)
 
+## Cryptography
+ * [cox](https://github.com/didactic-drunk/cox) - Crystal wrapper for the libsodium crypto API
```

Comment:
Do you plan to maintain a fork?

Reply:
Yes.  I'm currently giving the author more time before deforking the project.  Another Nacl project popped up with new code within the past few days so I'm trying to get this listed to avoid duplicated more effort.

Reply:
I've deforked and am taking over the project.  The last commit was over a year ago and he hasn't responded to any of my issues/PR's in a month.

Reply:
Ok, sounds good. What do you think about adding a link to the original repo for credits?

Reply:
If you think it will help people transition that's fine, happy to add it, but I'm not sure if there are even 1-5 users as the original library only included public key encryption.  No [symmetric key encryption, hmac, password gen, or any other features].

If it's for attribution I'm going to leave 2 places in the README.md with his name.
1. At the bottom he has himself listed as the creator maintainer.  I've changed it to "former creator, maintainer" and added myself as the last entry of contributors.
2. TBD near the top listing reasons for the fork with a link to the former library.

I've never had reason to fork a project before, but I need this for work, so it has to be maintained to a working minimum.  Whatever you suggest.

Reply:
Current state looks good to me.